tests/testthat/test-half.life.R

```diff
@@ -333,4 +333,22 @@ test_that("get_halflife_points", {
     suppressMessages(get_halflife_points(o_nca)),
     regexp = "More than one half-life calculation was attempted on the following rows: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11"
   )
+
+  # Half-life points guess right even if lambda.z.time.last != tlast

```

Comment:
Two notes on the comment here:

1. Please change "guess" to "selects" (it's not guessing 😄 )
2. Please put the original issue number in parentheses at the end to assist with tracking the context like `(#448)`